hintman.cabal

```diff
@@ -24,9 +24,17 @@ library
   hs-source-dirs:      src
   exposed-modules:     Hintman
                        Prelude
-
-  build-depends:       base-noprelude
-                     , relude
+                       Hintman.App
```

Comment:
Maybe indent these to match https://github.com/kowainik/hintman/blob/master/hintman.cabal#L27?